CONP_main_data_documentation.htm

```diff
@@ -0,0 +1,27 @@
+
+
+<h1>Uploading data for CONP</h1>
+
+<h2>1. Metadata</h2>
+
+We use the DATS.json file format to store metadata about CONP datasets. DATS.json is a flexible machine-readable structure allowing for sophisticated representation of a wide range of different types of datasets, from neuroimaging to genomics.  A DATS.json file must be prepared for each dataset.
+
+The CONP DATS.json schema includes the following required fields:
+
+- Dataset title and description
+- Creators of the dataset
+- DOI or other unique identifier
+- License under which the dataset is distributed
+- Keywords and specification of data type
+
+
+The schema also includes many other recommended and optional fields.  Full details can be found <a href="../DATS_fields.xlsx">here</a>.  The example DATS.json file for the 1000 Genomes Project <a href="https://github.com/emmetaobrien/1000GenomesProject/blob/master/DATS.json">here</a> can be used as a template.  An interface allowing users to fill in required fields online is under development <a href="https://dats-creator.herokuapp.com/">here</a>.
```

Comment:
A quick mention of `dataset_schema.json` would be helpful for those who are looking for structure guidance for properties.

Reply:
Sure, that makes sense and I will do it now.